client/state/user-settings/thunks/save-unsaved-user-settings.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { fromApi } from 'calypso/state/data-layer/wpcom/me/settings';
+import {
+	clearUnsavedUserSettings,
+	saveUserSettingsSuccess,
+} from 'calypso/state/user-settings/actions';
+import wp from 'calypso/lib/wp';
+import getUnsavedUserSettings from 'calypso/state/selectors/get-unsaved-user-settings';
+
+const wpcom = wp.undocumented();
+
+/**
+ * Redux thunk which exclusively updates given unsaved user settings
+ *
+ * @param {Array} fields Array of keys to be saved from unsaved user settings
+ */
+const saveUnsavedUserSettings = ( fields = [] ) => async ( dispatch, getState ) => {
```

Comment:
I opted for a thunk in this case because `<Account />` contains two separate forms (as of #48710) which the `userSettings` reducer is not capable of handling (and shouldn't be imo). That means we continue to manage state like `isSubmittingForm` etc within the `<Account />` component, for now. This may change with the PRs which should follow to clean things up a bit more.